style.css

```diff
@@ -0,0 +1,91 @@
+	
+	table tr td,table tr td{
```

Comment:
Please check the formatting on this file... something seems to have gone wrong.

Reply:
I manage to fix the duplication.
	this code is was selecting two titles that are the same.table tr td,table tr td{